app/layouts/DashboardLayout/SyncStatus.view/SyncStatus.view.tsx

```diff
@@ -82,7 +80,7 @@ const SyncStatus: FC = () => {
 
     statusCode = isSynced ? SYNCED : SYNCING;
   }
-  ipcRenderer.send('sync-status', statusCode);
+  sendSyncStatus(statusCode);
```

Comment:
How come we're sending a status code up, is this not something that should be managed further up the chain?

Reply:
It may appear as if its being passed up but it's actually inside DashboardLayout which wraps every route rendered. Once the user is authenticated SyncStatus runs before even displaying the component so I'm not sure of a better location.